notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -29,7 +29,7 @@
         "id": "eBEO2w9My9py"
```

Comment:
<p>Heading in sentence case.</p><p>Replace 'master' branch with 'main'</p><p> </p><p>Open in Cloud Notebook -&gt; Open in Vertex AI Workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>